components/pipeline-service/production/base/update-tekton-config-performance.yaml

```diff
@@ -31,6 +31,13 @@
   path: /spec/pipeline/options/deployments/tekton-operator-proxy-webhook/spec/replicas
   # default pipeline-service setting is 1
   value: 2
+- op: replace
+  path: /spec/pruner
+  value:
+    keep: 10
+    resources:
+      - pipelinerun
+    schedule: 0 * * * *
```

Comment:
should be once an hour per https://crontab.guru/every-1-hourhttps://crontab.guru/every-1-hour